UI/payments/payment2.html

```diff
@@ -161,14 +166,8 @@
                                  id="checkbox_all"
                                  data-dojo-type="dijit/form/Button">
                                 [% column.text -%]
-                                <script type="dojo/on" data-dojo-event="click">
-                                    require(["dojo/domReady!"], function(){
-                                        dojo.query('input[name^="checkbox_"]', this.domNode).forEach(function (node, index, arr) {
-                                            var n = dijit.getEnclosingWidget(node);
-                                            n.set('checked',!n.get('checked'));
-                                        });
-                                    });
-                                </script>
+                                <!--<script type="dojo/on" data-dojo-event="click"
+                                        src="lsmb/html/payments2_html"></script>-->
```

Comment:
Same about the "htmls", but also: maybe we can remove the commented out bit? In the new situation it doesn't add much value, I think.